screen2.html

```diff
@@ -0,0 +1,451 @@
+<!DOCTYPE html>
```

Comment:
give meaningful names for the files screen2 and screen3. 
in screen2 and screen3 also please fix all my comments that are present in index.html (mainly naming conventions and proper usage of tags)